### DISCUSSION TITLE
Incorrect characters after update

### DISCUSSION THREAD

**Discussion by AcidSoap** [Puharesource/TitleManager#345] [Q&A] [answered]
**Describe the bug**
After I updated TitleManager I have incorrect characters. I tried editing the default config file (Didn't create a new one with my own characters) and got the same problem.

**Screenshots**
Below are two screenshot with a before and after shot.

**Server information:**
 - TitleManager version: [Before 2.0.9 After 2.3.1 ]
 - Spigot version: [Example: Spigot 1.12.2]

https://i.imgur.com/O1BbLYy.png
https://i.imgur.com/4vr55ZG.png

**Comment by Puharesource** [comment, marked as answer]
Take a look at https://tmdocs.tarkan.dev/admins/troubleshooting for a fix.
